testsuite/fmi-cross-check/README.md

```diff
@@ -0,0 +1,51 @@
+# Run fmi-cross-check with OMSimulator
+The [fmi-cross-check repository]( https://github.com/modelica/fmi-cross-check)
+contains exported FMUs and results from various tools that take part in the FMI
+Cross-Check.
+
+To validate the results [FMPy](https://github.com/CATIA-Systems/FMPy) is used.
+
+
+## Dependencies
+  - OMsimulator in your PATH
+  - Python3 with modules
+    - OMSimulator
+    - pandas
+  - FMPy Docker
+
+
+## Run the test
+Just call the makefile and thats it.
+
+```bash
+$ make all
+```
+
+## Results
+For version `v2.1.0-dev-191-g470bf44-linux` on Ubuntu 18.04 we got:
+```bash
+        # ############################################# #
+        #             Final results summary             #
+        #       Total FMUs tested:              83      #
+        #       FMUs simulated succesfully:     59 / 83 #
+        #       FMUs verified:                  8 / 83  #
+        # ############################################# #
+```
+
+Also a detailed report will be saved `fmi-cross-check/OMSimulator_Results.csv`
+containing log and error messages.
+
+## Trouble shooting
+  - FMPy is missing.
+    - If you don't have FMPy installed you can use a [Docker image](https://hub.docker.com/repository/docker/anheuermann/fmpy) with FMPy.
+    Change `FMPY_DOCKER` in Makefile accordingly.
+  - Python can't find module `OMSimulator`.
+      - Download the latest pre-compiled binary for your system from
+        https://test.openmodelica.org/jenkins/job/OMSimulator/job/master/lastSuccessfulBuild/artifact/
+        and add the path to `OMSimulator.py` in `addResults.py`, e.g.:
+        ```python
+        sys.path.append("/opt/OMSimulator/lib")
+        from OMSimulator import OMSimulator
+        ```
```

Comment:
I tried to follow your instructions, but OMSimulator still cannot be found:

```
Traceback (most recent call last):
  File "addResults.py", line 410, in <module>
    df = runTest(fmiCrossCheckDir, platform, omsVersion)
  File "addResults.py", line 322, in runTest
    (result, output, error, resultComments, resultsCorrect) = addResult(crossCheckDir, fmiVersion, fmiType, platform, "OMSimulator", omsVersion, exportingToolID, exportingToolVersion, fmu)
  File "addResults.py", line 289, in addResult
    (resultsCorrect, resultDiff2) = diffSimulationResults(resultFile, referenceResult, Model_Name, testFMUDir, resultDir, relTol, relTol)
  File "addResults.py", line 114, in diffSimulationResults
    oms = OMSimulator()
  File "/home/lochel/OMSimulator/lib/OMSimulator.py", line 6, in __init__
    self.obj=cdll.LoadLibrary("libOMSimulator.so")
  File "/usr/lib/python3.8/ctypes/__init__.py", line 451, in LoadLibrary
    return self._dlltype(name)
  File "/usr/lib/python3.8/ctypes/__init__.py", line 373, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libOMSimulator.so: cannot open shared object file: No such file or directory
```

Reply:
Are you on Linux?
`libOMSimulator.so` has to lie next to `OMSimulator.py` in the directory you added with 
```python
sys.path.append("/opt/OMSimulator/lib")
from OMSimulator import OMSimulator
```

I'm using the binary from https://test.openmodelica.org/jenkins/job/OMSimulator/job/master/374/artifact/OMSimulator-linux-amd64-v2.1.0-dev-181-g524ed01.tar.gz and copied them to `/opt/OMSimulator-dev-181/lib`.
See [addResults.py line 9](https://github.com/OpenModelica/OMSimulator/pull/779/files#diff-308208c1d22eb636ec119472b3c224aeR9).
The OMSimulator Python module build with OpenModelica is currently not working.

Reply:
Yes, it is linux and `libOMSimulator.so` is next to `OMSimulator.py`. I am not using the OpenModelica build, but the artefact you're pointing to.

Reply:
https://github.com/OpenModelica/OMSimulator/pull/780 should fix your problem.